Tests/OpenCombineTests/ObservableObjectTests.swift

```diff
@@ -0,0 +1,423 @@
+            counter += 1
+        }.store(in: &disposeBag)
+
+        XCTAssertEqual(observableObject.publishedValue0, 0)
+        XCTAssertEqual(observableObject.simpleValue, "what")
+        XCTAssertEqual(observableObject.subclassPublished0, 0)
+        XCTAssertEqual(observableObject.subclassPublished1, 1)
+        XCTAssertEqual(observableObject.subclassPublished2, 2)
+
+        observableObject.publishedValue0 += 5
+
+        XCTAssertEqual(counter, 1)
+        XCTAssertEqual(observableObject.publishedValue0, 5)
+
+        Published<String>[_enclosingInstance: observableObject,
+                          wrapped: \.simpleValue,
+                          storage: \.publishedValue1] += "???"
+
+        XCTAssertEqual(counter, 2)
+        XCTAssertEqual(observableObject.simpleValue, "what")
+
+        observableObject.subclassPublished0 += 3
+
+        XCTAssertEqual(counter, 3)
+        XCTAssertEqual(observableObject.subclassPublished0, 3)
+
+        observableObject.subclassPublished1 += 3
+
+        XCTAssertEqual(counter, 4)
+        XCTAssertEqual(observableObject.subclassPublished1, 4)
+
+        observableObject.subclassPublished2 += 3
+
+        XCTAssertEqual(counter, 5)
+        XCTAssertEqual(observableObject.subclassPublished1, 4)
+    }
+
+    func testObjCClassSubclass() {
+        let observableObject = ObjCClassSubclass()
+        let publisher1 = observableObject.objectWillChange
+        let publisher2 = observableObject.objectWillChange
+        XCTAssert(publisher1 === publisher2)
+    }
+
+    func testResilientClassSubclass() {
+        let observableObject = ResilientClassSubclass()
+        let publisher1 = observableObject.objectWillChange
+        let publisher2 = observableObject.objectWillChange
+
+        XCTAssert(publisher1 === publisher2)
+    }
+
+    func testResilientClassSubclass2() {
+        let observableObject = ResilientClassSubclass2()
+        let publisher1 = observableObject.objectWillChange
+        let publisher2 = observableObject.objectWillChange
+
+        XCTAssert(publisher1 === publisher2)
+    }
+
+    func testGenericClass() {
+        let observableObject = GenericClass(123, true)
+
+        var counter = 0
+
+        observableObject.objectWillChange.sink { counter += 1 }.store(in: &disposeBag)
+        XCTAssertEqual(counter, 0)
+        XCTAssertEqual(observableObject.value1, 123)
+        XCTAssertEqual(observableObject.value2, true)
+
+        observableObject.value1 += 1
+
+        XCTAssertEqual(counter, 1)
+        XCTAssertEqual(observableObject.value1, 124)
+
+        observableObject.value2.toggle()
+
+        XCTAssertEqual(counter, 2)
+        XCTAssertEqual(observableObject.value2, false)
+    }
+
+    func testGenericSubclassOfResilientClass() {
+        let observableObject = ResilientClassGenericSubclass("hello", true)
+
+        var counter = 0
+
+        // A bug in Combine (FB7471594). It should not crash. Why would it crash?
+        assertCrashesOnDarwin {
```

Comment:
This bug has been fixed, it doesn't crash anymore. Let's remove this.

Reply:
Hmm, it seems this bug is not fixed on macOS 10.15.7. Which version are you using?

Reply:
The latest iOS. Combine on macOS is always a little bit behind its iOS version.

Reply:
So should we run this test suit on only iOS?

Reply:
When testing against Combine — yes. But this applies to all the tests, not just this.

Reply:
I addressed 👍